setup.cfg

```diff
@@ -31,13 +31,13 @@ include_package_data = True
 code_style =
     pre-commit~=2.12
 myst =
-    myst-nb>=0.11,<0.13
+    myst-nb>=0.12,<0.14
 rtd =
     myst-parser
     sphinx-book-theme
 testing =
     coverage<5.0
-    jupyter-book>=0.11.0a1,<0.12
+    jupyter-book>=0.11,<0.12
```

Comment:
@AakashGfude this is an issue as `jupyter-book` wont be updated to include `sphinx4` compatibility until this package is checked and updated :-)

Reply:
```
▶ grep -r "jupyter-book" .       
./sphinx_jupyterbook_latex.egg-info/requires.txt:jupyter-book<0.12,>=0.11.0a1
./CHANGELOG.md:1. Make this package a "standalone" sphinx extension, independent of jupyter-book, i.e. not dependant on the presence of a `_toc.yml`.
./CHANGELOG.md:This is the first release of this package. And includes a number of features, improvements and bug fixes for pdf builds in jupyter-book projects.
Binary file ./tests/roots/test-partsToc/_build/.doctrees/environment.pickle matches
./docs/intro.md:**Improvements to LaTeX for [jupyter-book](https://jupyterbook.org/intro.html)**.
./docs/intro.md:extension to handle LaTeX builds for [jupyter-book](https://jupyterbook.org/)
./docs/intro.md:  using `myst_amsmath_enable` key of `jupyter-book`.
./.github/workflows/pdf.yml:        git clone https://github.com/executablebooks/jupyter-book.git
./.github/workflows/pdf.yml:        cd jupyter-book
./.github/workflows/pdf.yml:        path: jupyter-book/docs/_build/latex/book.pdf
./setup.cfg:    jupyter-book>=0.11,<0.12
```

Reply:
looks like this requirement could be removed?

Reply:
ah I see

```
ImportError while importing test module '/Users/matthewmckay/repos-collab/ebp/sphinx-jupyterbook-latex/tests/test_jupyter_book.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_jupyter_book.py:4: in <module>
    from jupyter_book.cli.main import build
E   ModuleNotFoundError: No module named 'jupyter_book'
```

Reply:
can we update these tests to run using `sphinx_build` rather than `jupyter-book`?

Reply:
@chrisjsewell is there any way to import these fixtures to our projects so we aren't replicating the code across our test suites. 

https://github.com/executablebooks/MyST-NB/blob/448d36f483de2bdbf2ac71d8a9b3e8fe75918be7/tests/conftest.py#L152

Maybe we need a `jupyterbook-testing` package or something like that?

Reply:
I see there is some custom code in each of these for context though so maybe harder that it looks initially.